### PR TITLE
New `DashModel.refreshContextModel`

### PR DESCRIPTION
**Pull request by cnrudd**
…loadSupport enabled views directly.

Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry.
- [x] Reviewed for breaking changes: not a breaking change.
- [x] Updated doc comments / prop-types: not required.
- [x] Reviewed and tested on Mobile: not required.
- [x] Created Toolbox branch: yes! PR: https://github.com/xh/toolbox/pull/663

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Review by TomTirapani — APPROVED** [review]
Looks great to me 👍

File: desktop/cmp/dash/DashModel.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export abstract class DashModel<VSPEC, VSTATE, VMODEL> extends HoistModel {
     @observable.ref state: VSTATE[];
     @managed @observable.ref viewModels: VMODEL[] = [];
 
+    @managed refreshContextModel: RefreshContextModel;
```
Comment (ghsolomon) [inline desktop/cmp/dash/DashModel.ts:21]: Can make this `readonly`

**Review by ghsolomon — APPROVED** [review]
Looks good! Just one very small comment

File: desktop/cmp/dash/DashModel.ts (hunk 1 above)
Comment (cnrudd) [reply desktop/cmp/dash/DashModel.ts]: Done.  Thank you!

**Pull request merged**